### PR TITLE
[FIX] Login failure after multiple logouts

### LINKED ISSUE #58: Logout Failure Issue in iframe after logout from rc4git
When a user logins for the first time, iframe login works fine, but if the user logs out and then try to login immediately then the iframe login fails and would require the user to logout again to work.
In short for every login after one logout, it requires the user to logout and then login again.

https://user-images.githubusercontent.com/51796498/109124774-52420400-7771-11eb-9c18-6588497b584c.mp4

Iframe logout from client side solves this issue.

### PR DESCRIPTION
**Pull request by dhruvjain99**
https://user-images.githubusercontent.com/51796498/109427186-3495c880-7a17-11eb-9bd1-f3d3c48397ab.mp4

Fixes #58

### CODE REVIEW

File: client/src/components/SignedLeftSidebar/index.js [hunk 1]
```diff
@@ -181,6 +178,15 @@ export default function SignedLeftSidebar(props) {
   };
 
   const logout = () => {
+    const iframe = window.document.getElementsByTagName("iframe")[0];
```
Comment (RonLek) [inline client/src/components/SignedLeftSidebar/index.js:181]: I think here `document.getElementsByTagName` should work just as well. Cmiiw!

File: client/src/components/SignedLeftSidebar/index.js [hunk 2]
```diff
@@ -181,6 +178,15 @@ export default function SignedLeftSidebar(props) {
...
+    if (iframe) {
+      iframe.contentWindow.postMessage(
+        { externalCommand: "logout" },
+        `${rcApiDomain}`
+      );
+      window.document.getElementsByClassName("loading-chatWindow")[0].style =
```
Comment (RonLek) [inline client/src/components/SignedLeftSidebar/index.js:187]: `document.getElementsByClassName` here as well.

File: client/src/components/SignedLeftSidebar/index.js [hunk 3]
```diff
@@ -205,6 +211,8 @@ export default function SignedLeftSidebar(props) {
         console.log("Error logging out --->", err);
         loadingIcon.classList.add("hide-logout-loading");
         logoutButton.classList.remove("disable-click");
+        window.document.getElementsByClassName("loading-chatWindow")[0].style =
```
Comment (RonLek) [inline client/src/components/SignedLeftSidebar/index.js:214]: Same for this one.

**Comment by dhruvjain99** [comment]
Created the same with updates changes as requested from the fork.

**Pull request closed without merging**